### PR TITLE
Timeout should actually wrap test logic

### PR DESCRIPTION
**Pull request by xbrianh**
Tested on the integration test with 6 second timeouts.

### CODE REVIEW

File: tests/integration/test_end_to_end_dcp.py [hunk 1]
```diff
@@ -104,9 +104,8 @@ class TestSmartSeq2Run(TestEndToEndDCP):
     ]
 
     def test_smartseq2_run(self):
-        runner = self.ingest_store_and_analyze_dataset(dataset_fixture='Smart-seq2')
-
-        with Timeout(110 * 60) as to:  # timeout after 1 hour and 50 minutes
+        with Timeout(6) as to:  # timeout after 1 hour and 50 minutes
```
Comment (parthshahva) [inline tests/integration/test_end_to_end_dcp.py]: @xbrianh I'm guessing you want to change this back to 110 * 60?

File: tests/integration/test_end_to_end_dcp.py [hunk 2]
```diff
@@ -140,9 +142,8 @@ class Test10xRun(TestEndToEndDCP):
     ]
 
     def test_10x_run(self):
-        runner = self.ingest_store_and_analyze_dataset(dataset_fixture='10x')
-
-        with Timeout(110 * 60) as to:  # timeout after 1 hour and 50 minutes
+        with Timeout(6) as to:  # timeout after 1 hour and 50 minutes
```
Comment (parthshahva) [inline tests/integration/test_end_to_end_dcp.py]: @xbrianh change the timeout value here as well?

File: tests/integration/test_end_to_end_dcp.py [hunk 3]
```diff
@@ -118,6 +117,9 @@ def test_smartseq2_run(self):
             except:
                 pass
 
+        print()
```
Comment (parthshahva) [inline tests/integration/test_end_to_end_dcp.py]: empty print?

File: tests/integration/test_end_to_end_dcp.py [hunk 4]
```diff
@@ -154,6 +155,9 @@ def test_10x_run(self):
...
```
Comment (parthshahva) [inline tests/integration/test_end_to_end_dcp.py]: empty print?

File: tests/integration/test_end_to_end_dcp.py (hunk 1 above)
Comment (xbrianh) [reply tests/integration/test_end_to_end_dcp.py]: fixed, thanks

File: tests/integration/test_end_to_end_dcp.py (hunk 2 above)
Comment (xbrianh) [reply tests/integration/test_end_to_end_dcp.py]: fixed

File: tests/integration/test_end_to_end_dcp.py (hunk 3 above)
Comment (xbrianh) [reply tests/integration/test_end_to_end_dcp.py]: This makes GitLab log output prettier (it was squished into another line for some reason).

**Pull request merged**